python/deltalake/table.py

```diff
@@ -4,10 +4,41 @@
 import pyarrow
 from pyarrow.dataset import dataset
 
-from .deltalake import RawDeltaTable
+from .deltalake import RawDeltaTable, RawDeltaTableMetaData
 from .schema import Schema, pyarrow_schema_from_json
 
 
+class Metadata:
```

Comment:
@houqp: I was wondering if I could change this class (for documentation purposes) to something like this but I need your opinion on that:

```
class Metadata:
    """ Create a Metadata instance. """

    def __init__(self, table: RawDeltaTable):
        self._metadata = table.metadata()

    @property
    def id(self):
        """ Return the unique identifier of the DeltaTable. """
        return self._metadata.id

    @property
    def name(self):
        """ Return the user-provided identifier of the DeltaTable. """
        return self._metadata.name

    @property
    def description(self):
        """ Return the user-provided description of the DeltaTable. """
        return self._metadata.description

    @property
    def configuration(self):
        """ Return the DeltaTable properties. """
        return self._metadata.configuration

    @property
    def partition_columns(self):
        """ Return an array containing the names of the partitioned columns of the DeltaTable. """
        return self._metadata.partition_columns
```

Reply:
sounds like a good idea :+1:

Reply:
Ok I will add this in this PR :)